src/main/java/com/brandon3055/draconicevolution/common/items/tools/Magnet.java

```diff
@@ -25,13 +26,15 @@
 import com.brandon3055.draconicevolution.common.items.ItemDE;
 import com.brandon3055.draconicevolution.common.lib.References;
 
+import baubles.api.BaubleType;
+import baubles.api.IBauble;
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 
 /**
  * Created by brandon3055 on 9/3/2016.
  */
-public class Magnet extends ItemDE {
+public class Magnet extends ItemDE implements IBauble {
```

Comment:
Please, implement `IBauble` as optional interface. No need in creation of hard dependency.